tests/integration/marketplacesuite.nim

```diff
@@ -45,25 +45,40 @@ template marketplacesuite*(name: string, body: untyped) =
     proc periods(p: int): uint64 =
       p.uint64 * period
 
-    proc createAvailabilities(datasetSize: int, duration: uint64) =
+    # Purposely overshooting
+    # TODO: find a better way to compute slot size for the given
+    # dataset size
+    # I found this snippet in tests/integration/testproofs.nim:
+    # let datasetSizeInBlocks = 3
+    # let data = await RandomChunker.example(blocks=datasetSizeInBlocks)
+    # # original data = 3 blocks so slot size will be 4 blocks
+    # let slotSize = (DefaultBlockSize * 4.NBytes).Natural.u256
+    proc availabilityTotalSizeForDataSize(dataSize: int): UInt256 =
+      (dataSize * 2).u256
+
+    proc createAvailabilitiesForData(
+        data: string, duration: uint64, minPricePerBytePerSecond: UInt256
+    ) =
+      let availabilityTotalSize = availabilityTotalSizeForDataSize(data.len)
+      let totalCollateral = availabilityTotalSize * minPricePerBytePerSecond
```

Comment:
I think this should be `availabilityTotalSize * collateralPerByte`.

Reply:
Yes, thanks. Updated other dependent places as well.